stable/prometheus-operator/crds/crd-podmonitor.yaml

```diff
@@ -236,5 +253,17 @@ spec:
           - podMetricsEndpoints
           - selector
           type: object
+      required:
+      - spec
       type: object
   version: v1
+  versions:
+  - name: v1
+    served: true
+    storage: true
+status:
+  acceptedNames:
+    kind: ""
+    plural: ""
+  conditions: []
+  storedVersions: []
```

Comment:
Why is status now part of the yaml? Status is supplied by Kubernetes.
When applying an update it will fail as the status does not match.

Reply:
Looks like the source of this is here: https://github.com/coreos/prometheus-operator/blob/master/example/prometheus-operator-crd/monitoring.coreos.com_alertmanagers.yaml#L4502-L4507